docs/markdown/Python-module.md

```diff
@@ -101,6 +101,11 @@ the addition of the following:
   `/usr/lib/site-packages`. When subdir is passed to this method,
   it will be appended to that location. This keyword argument is
   mutually exclusive with `install_dir`
+- `limited_api`: *since 1.3.0* A string containing the Python version
+  of the [Py_LIMITED_API](C-https://docs.python.org/3/c-api/stable.html) that
```

Comment:
This link looks odd, is the `C-` part a typo?

Reply:
Ah yes, this is definitely a typo on my part. I've fixed this.